apps/web/src/utils/proofs/sybil_resistance.ts

```diff
@@ -86,25 +86,29 @@ async function signMessageWithTrustedSigner(
   targetAddress: Address,
   expiry: number,
 ) {
-  const account = privateKeyToAccount(`0x${trustedSignerPKey}`);
-
   // encode the message
   const message = encodePacked(
-    ['bytes2', 'address', 'address', 'address', 'uint256'],
+    ['bytes2', 'address', 'address', 'address', 'uint64'],
     ['0x1900', targetAddress, trustedSignerAddress, claimerAddress, BigInt(expiry)],
   );
 
   // hash the message
   const msgHash = keccak256(message);
 
   // sign the hashed message
-  const signature = await account.signMessage({ message: msgHash });
+  const { r, s, v } = await sign({
+    hash: msgHash,
+    privateKey: `0x${trustedSignerPKey}`,
+  });
+
+  // combine r, s, and v into a single signature
+  const signature = `${r.slice(2)}${s.slice(2)}${(v as bigint).toString(16)}`;
```

Comment:
I know you'd padded v to two chars in an earlier implementation--why the change?

Reply:
did not seem necessary, but can go ahead and add back if that's safer 👍 my understanding is its typically either 27n or 28n.